backend/inspirehep/serializers.py

```diff
@@ -82,4 +86,25 @@ def serialize_search(self, pid_fetcher, search_result, **kwargs):
             facets. This is used with
             ``inspirehep.search.factories.search.search_factory_only_with_aggs``.
         """
+
+        search_result["aggregations"] = self._unnest_aggregations(
+            search_result.get("aggregations", {})
+        )
+
         return json.dumps(search_result)
+
+    def _unnest_aggregations(self, aggregations):
```

Comment:
should be decorated with `@staticmethod`, as you don't use `self` in the body of the function.